src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimantResponseService.java

```diff
@@ -74,8 +78,26 @@ public void save(
         claimantResponseRule.assertCanBeRequested(claim, claimantId);
 
         Claim updatedClaim = caseRepository.saveClaimantResponse(claim, claimantResponse, authorization);
+
         claimantResponseRule.isValid(updatedClaim);
         formaliseResponseAcceptance(claimantResponse, updatedClaim, authorization);
+
+        Response response = claim.getResponse().orElseThrow(IllegalStateException::new);
+
+        if(isFullDefenseDisputeAcceptation(response, claimantResponse)) {
```

Comment:
Please download the styling configuration from https://tools.hmcts.net/confluence/display/ROC/Developer+onboarding and apply it to your code before pushing